src/Workspaces/CoreTest/SymbolKeyTests.cs

```diff
@@ -736,9 +736,9 @@ private void TestRoundTrip(IEnumerable<ISymbol> symbols, Compilation compilation
 
         private void TestRoundTrip(ISymbol symbol, Compilation compilation, Func<ISymbol, object> fnId = null)
         {
-            var id = SymbolKey.ToString(symbol);
+            var id = SymbolKey.CreateString(symbol);
             Assert.NotNull(id);
-            var found = SymbolKey.Resolve(id, compilation).GetAnySymbol();
+            var found = SymbolKey.ResolveString(id, compilation).GetAnySymbol();
```

Comment:
Could you please add a test for the case where version number doesn't match?

Reply:
sure